user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/Logging/Dashboards_and_Low_Code_Apps_Logging.md

```diff
@@ -0,0 +1,50 @@
+
+```md
+| Setting Name                  | Setting Type | Default Value        | Description                                                                                                                      |
+| ----------------------------- | ------------ | -------------------- | -------------------------------------------------------------------------------------------------------------------------------- |
+| EnableDebugLogging            | Boolean      | false                | Enables or disables the logging of debug information.                                                                            |
+| EnableAllowedOperationLogging | Boolean      | false                | Enables or disables the logging of WAF information.                                                                              |
+| EnableClientMetricLogging     | Boolean      | true                 | Enables or disables the logging of client metrics.                                                                               |
+| MaxFileSize                   | Number       | 52428800 (50MB)      | Maximum size of a single log file.                                                                                               |
+| FilesToKeep                   | Number       | 14                   | Number of files to keep for each type of log.                                                                                    |
+| RollingInterval               | String       | Day                  | Time period included in each log file.                                                                                           |
+| AsyncBufferSize               | Number       | 10000                | Size of the buffer of the worker writing the log file to disk.                                                                   |
+| BlockWhenAsyncBufferIsFull    | Boolean      | false                | If true, the logger will wait until there is room in the buffer; otherwise, log entries will be dropped when the buffer is full. |
+| HealthCheckInterval           | Number       | 2500                 | Number of log entries before a health check of the logger is performed.                                                          |
+| FlushToDiskInterval           | Timespan     | 00:00:01             | Timespan between the log being written to memory and the result being flushed to disk.                                           |
+| Filename                      | String       | web.log              | Filename of the WebAPI log files.                                                                                                |
+| ClientMetricFilename          | String       | web.clientmetric.log | Filename of the client metric log files.                                                                                         |
+```
+
+> [!NOTE]
+> Before DataMiner 10.1.12/10.2.0, the config file was named `SLWAF.conf`.
```

Comment:
If I understand correctly, the web logging folder is only available from DataMiner 10.3.0 [CU14]/10.4.0 [CU2]/10.4.5 onwards (RN 39000). If a logging config file is already available in earlier versions, where is this located and what is it used for?

Reply:
The WebAPI logging already existed since 10.2.0/10.1.8, this RN adds the specific logging for client metrics. The config file is still in the same location & has the same function. It is now just shared for the 2 logging types.